code/modules/SCP/SCPs/SCP-409.dm

```diff
@@ -0,0 +1,39 @@
+/obj/structure/scp409/
+	name = "SCP-409"
+	desc = "A white crystal, something beckons you to come and touch it..."
+	icon = 'icons/SCP/scp-409.dmi'
+	icon_state = "409"
+	layer = 2.9
+	anchored = TRUE // makes it not able to be moved
+	density = TRUE // makes it have collisions
```

Comment:
why so many useless comments

Reply:
new coder friendliness :) also its just comments, why do you care?

Reply:
comment issues fixed